versioned_docs/version-24.10/graph-views/upgrading-map-legacy.md

```diff
@@ -0,0 +1,207 @@
+
+2. Update the Centreon MAP (Legacy) server:
+
+    ```shell
+    dnf update centreon-map-server
+    ```
+
+</TabItem>
+<TabItem value="Debian 12" label="Debian 12">
+
+```shell
+echo "deb https://packages.centreon.com/apt-standard-24.04-stable/ $(lsb_release -sc) main" | tee /etc/apt/sources.list.d/centreon.list
+echo "deb https://packages.centreon.com/apt-plugins-stable/ $(lsb_release -sc) main" | tee /etc/apt/sources.list.d/centreon-plugins.list
+```
+
+> Install the Centreon Business repository. You can find it on the [support portal](https://support.centreon.com/hc/en-us/categories/10341239833105-Repositories).
+
+2. Update Centreon MAP (Legacy) server:
+
+    ```shell
+    apt install --only-upgrade centreon-map-server
+    ```
+
+</TabItem>
+</Tabs>
+
+3. Enable and start the **centreon-map** service:
+
+    ```shell
+    systemctl enable centreon-map
+    systemctl start centreon-map
+    ```
+
+4. This point only applies if you customized your **centreon-map.conf** configuration file. When upgrading your MAP (Legacy) module, the **/etc/centreon-studio/centreon-map.conf** file is not upgraded automatically; the new configuration file brought by the rpm does not replace the old file. You must copy the changes manually to your customized configuration file.
+
+  * The old configuration file is renamed **centreon-map.conf.rpmsave**
+  * The upgrade installs a new **centreon-map.conf** file.
+
+  Run a diff between the old and the new configuration files:
+
+  ```shell
+  diff -u /etc/centreon-studio/centreon-map.conf /etc/centreon-studio/centreon-map.conf.rpmsave
+  ```
+
+  For each difference between the files, assess whether you should copy it from **centreon-map.conf.rpmsave** to **centreon-map.conf**.
+
+## Step 2: Centreon MAP (Legacy) web interface
+
+<Tabs groupId="sync">
+<TabItem value="Alma / RHEL / Oracle Linux 8" label="Alma / RHEL / Oracle Linux 8">
+
+```shell
+dnf update centreon-map-web-client
+```
+</TabItem>
+<TabItem value="Alma / RHEL / Oracle Linux 9" label="Alma / RHEL / Oracle Linux 9">
+
+```shell
+dnf update centreon-map-web-client
+```
+
+</TabItem>
+<TabItem value="Debian 11 & 12" label="Debian 11 & 12">
+
+```shell
+apt install --only-upgrade centreon-map-web-client
+```
+
+</TabItem>
+</Tabs>
+
+Complete the upgrade: 
+1. Go to **Administration > Extensions > Manager**.
+2. Search for **Map web client**.
+3. Click the update button (module & widget parts).
+
+## Step 3: Centreon MAP (Legacy) desktop client
+
+If the user's computer has an online connection, the desktop client is
+automatically upgraded to the latest version that corresponds to the server.
+
+Alternatively, the client can be downloaded through the `Monitoring >
+Map` menu and the **Desktop client** button.
+
+## Step 4: Update dialects in .properties files
+
+In the **/etc/centreon-studio/centreon-database.properties** and the **/etc/centreon-studio/studio-database.properties** files, replace **MySQL5Dialect** with **MariaDB10Dialect**.
```

Comment:
MariaDB10Dialect -> MariaDBDialect

Reply:
But I think the package will just do it automatically on upgrade... so we need to double-check if it's still worth mentioning this.